examples/ompt/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -fopenmp")
+add_executable(hello_world hello_world.c)
```

Comment:
I think this should be wrapped into a `if(HPXMP_WITH_OMPT)...endif()`. Otherwise it will be compiled even if no OMPT support is available.